src/components/profile/EditProfile.js

```diff
@@ -114,7 +122,14 @@ const styles = StyleSheet.create({
   saveButton: {
     position: 'absolute',
     top: 0,
-    right: 0
+    right: 0,
+    paddingHorizontal: normalize(16),
+    marginVertical: 0,
+    fontSize: normalize(14)
```

Comment:
If the contained `Text` component already has `fontSize`, this feels unnecessary.

Reply:
Done